source/install/install-rhel-7-postgresql.rst

```diff
@@ -44,9 +44,9 @@ Installing PostgreSQL Database
 
   ``psql``
 
-10.  Create the Mattermost database.
+10.  Create the Mattermost database with UTF-8 encoding.
```

Comment:
We don't need to mention the UTF-8 part here. The command is all the user needs to execute.